ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/BindingVerifier.kt

```diff
@@ -53,8 +56,16 @@ abstract class BindingVerifier {
                                 "\n$IDP_ERROR_RESPONSE_REMINDER_MESSAGE")
             }
         }
-    }
 
-    abstract fun verifyError()
-    abstract fun verify()
+        fun getBindingVerifier(response: Response): BindingVerifier {
+            return when (response.determineBinding()) {
+                SamlProtocol.Binding.HTTP_REDIRECT -> RedirectBindingVerifier(response)
+                SamlProtocol.Binding.HTTP_POST -> PostBindingVerifier(response)
+                else -> throw UnsupportedOperationException("Binding is not currently supported.")
+            }
+        }
```

Comment:
I put this method in the `BindingVerifier` companion object, which felt odd to me. I'm open to suggestions on where to put it.

Reply:
Going with Blen's suggestion.